meta/store.go

```diff
@@ -354,8 +353,8 @@ func (s *Store) createLocalNode() error {
 	return nil
 }
 
-// waitForLeader sleeps until a leader is found or a timeout occurs.
-func (s *Store) waitForLeader(timeout time.Duration) error {
+// WaitForLeader sleeps until a leader is found or a timeout occurs.
+func (s *Store) WaitForLeader(timeout time.Duration) error {
 	// Begin timeout timer.
 	timer := time.NewTimer(timeout)
```

Comment:
Probably should check to see if there's already a leader before starting a polling loop.